oidc-exchange.py

```diff
@@ -0,0 +1,156 @@
+import os
```

Comment:
Please change the filename to use an underscore, not a hyphen.  `oidc_exchange.py`

Reply:
This is an executable script, not an importable module, so I think the hyphenation is appropriate. See also `print-hash.py` in this repo 🙂.

Reply:
I was looking at why `pre-commit` was failing.
https://results.pre-commit.ci/repo/github/178055147

Reply:
+1 on keeping the hyphen, _specifically_ because it's not meant to be imported. This should not work. Maybe, when I get to refactoring this whole thing into a Python-based entry point, it'll make sense, but right now it's rather opposite.